torch/_torch_docs.py

```diff
@@ -7846,23 +7825,8 @@ def merge_dicts(*dicts):
            r"""
 round(input, *, out=None) -> Tensor
 
-Returns a new tensor with each of the elements of :attr:`input` rounded
-to the closest integer.
-
-Args:
-    {input}
-
-Keyword args:
-    {out}
-
-Example::
-
-    >>> a = torch.randn(4)
-    >>> a
-    tensor([ 0.9920,  0.6077,  0.9734, -1.0362])
-    >>> torch.round(a)
-    tensor([ 1.,  1.,  1., -1.])
-""".format(**common_args))
+Alias for :func:`torch.special.round`.
```

Comment:
Let's keep torch.round the source of truth to be most consistent with the Python Array API (https://data-apis.org/array-api/latest/API_specification/elementwise_functions.html?highlight=round#round-x) -- is our round equivalent to their round, by the way?

We can alias torch.special.round to torch.round

Reply:
Makes sense. Will keep the Array API in mind going forward. Thanks!

> is our round equivalent to their round, by the way?

Will confirm that!